client/wfm/README.md

```diff
@@ -29,19 +23,3 @@ In order to use this module, begin by installing all dependencies:
 If inside the [lerna](http://lernajs.io)-based mono repo: `lerna bootstrap --scope=@raincatcher/wfm --include-filtered-dependencies`
 
 For standalone usage: `npm install`
-
-For implementing a base flow, see the [examples folder](./examples/).
```

Comment:
I don't think we have step creation set in stone yet, I just removed the outdated decription that I had for the previous version with the names like `Process/ProcessInstance` etc, because their usage was to be different.